README.md

```diff
@@ -545,60 +574,70 @@ import ballerina/http;
 import ballerina/io;
 import ballerina/log;
 
-endpoint http:Client studentData {
-    url: " http://localhost:9292"
-};
+http:Client studentService = new("http://localhost:9292",
+config = { httpVersion: "2.0" });
 
 public function main() {
     http:Request req = new;
     int operation = 0;
-    while (operation != 6) {
-        // Print options menu to choose from.
-        io:println("Select operation.");
-        io:println("1. Add student");
-        io:println("2. View all students");
-        io:println("3. Delete a student");
-        io:println("4. Make a mock error");
-        io:println("5: Get a student's marks");
-        io:println("6. Exit \n");
-
-        // Read user's choice.
-        string choice = io:readln("Enter choice 1 - 5: ");
-        if (!isInteger(choice)) {
-            io:println("Choice must be of a number");
-            io:println();
-            continue;
-        }
-        operation = check <int>choice;
-        // Program runs until the user inputs 6 to terminate the process.
-        if (operation == 6) {
-            break;
-        }
-        if (operation == 1) {
-            // User chooses to add a student.
-            addStudent(req);
-        }  else if (operation == 2) {
-            // User chooses to list down all the students.
-            viewAllStudents();
-        } else if (operation == 3) {
-            // User chooses to delete a student by Id.
-            deleteStudent();
-        } else if (operation == 4) {
-            // User chooses to make a mock error.
-            makeError();
-        } else if (operation == 5){
-            // User chooses to get the marks of a particular student.
-          getMarks();
-        } else {
-            io:println("Invalid choice \n");
+        while (operation != 6) {
+            // Print options menu to choose from.
+            io:println("Select operation.");
+            io:println("1. Add student");
+            io:println("2. View all students");
+            io:println("3. Delete a student");
+            io:println("4. Make a mock error");
+            io:println("5: Get a student's marks");
+            io:println("6. Exit \n");
+
+            // Read user's choice.
+            string choice = io:readln("Enter choice 1 - 5: ");
+            if (!isInteger(choice)) {
+                io:println("Choice must be of a number");
+                io:println();
+                continue;
+            }
+
+            var intOp = int.convert(choice);
+
+            if (intOp is int){
+                operation = intOp;
+            }
+
+            // Program runs until the user inputs 6 to terminate the process.
+            if (operation == 6) {
```

Comment:
Can we use the match statment here? Refer https://ballerina.io/learn/by-example/match.html

Reply:
Yes I will make the necessary changes.